source/winVersion.py

```diff
@@ -28,38 +126,28 @@ def isUwpOcrAvailable():
 
 
 WIN10_VERSIONS_TO_BUILDS = {
-	1507: 10240,
-	1511: 10586,
-	1607: 14393,
-	1703: 15063,
-	1709: 16299,
-	1803: 17134,
-	1809: 17763,
-	1903: 18362,
-	1909: 18363,
-	2004: 19041,
-	2009: 19042,
+	"1507": 10240,
+	"1511": 10586,
+	"1607": 14393,
+	"1703": 15063,
+	"1709": 16299,
+	"1803": 17134,
+	"1809": 17763,
+	"1903": 18362,
+	"1909": 18363,
+	"2004": 19041,
+	"20H2": 19042,
 }
 
 
 def isWin10(version: int = 1507, atLeast: bool = True):
 	"""
+	@deprecated: use getWinVer().isWin10 instead.

```

Comment:
If we agree about this approach and I think it makes sense, we can as well remove this entirely as part of the 2021.1 release.